### DISCUSSION TITLE
What is brutalized  doing?

### DISCUSSION THREAD

**Discussion by wanderer** [Vectorized/solady#854] [General]
i dont understand

**Comment by joshuacheong** [comment]
What you mean bru

**Comment by wanderer** [comment]
here https://github.com/Vectorized/solady/blob/c6738e40225288842ce890cd265a305684e52c3d/test/utils/mocks/MockERC6551.sol#L9
But it is used through out the tests

**Comment by Vectorized** [comment]
It's for intentionally corrupting the upper 96 bits of `a`. 

Because we want to make sure that the assembly is agnostic to the dirty upper bits.

@pop-punk has some `_brutalized` merch.

**Reply by heyJonBray** [reply]
merch you say? 👀
